standard_risk_tests.ipynb

```diff
@@ -0,0 +1,1116 @@
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "column_labels = []\n",
+    "input_schema = model.get(\"modelMetaData\", {}).get(\"inputSchema\", [])\n",
+    "if len(input_schema) != 1:\n",
+    "\tdisplay(HTML(\"The dashboard model requires exactly one input schema to be specified. Please go to the model's schema page in MOC, and generate and edit a valid schema for the input data\"))\n",
+    "else:\n",
+    "\tinput_schema = model.get(\"modelMetaData\", {}).get(\"inputSchema\", [])\n",
+    "\tresponse = session.post(moc_url + \"model-manage/api/schema/validate/?extended=true\", json = input_schema[0].get(\"schemaDefinition\", {}))\n",
+    "\tif response.ok:\n",
+    "\t\tdisplay(HTML(\"The example model contains a valid schema\"))\n",
+    "\t\tfor col in input_schema[0].get(\"schemaDefinition\", {}).get(\"fields\", []):\n",
+    "\t\t\tcolumn_labels.append(col.get(\"name\", \"\"))\n",
+    "\t\tdisplay(column_labels)\n",
+    "\telse:\n",
+    "\t\tdisplay(HTML(\"The dashboard model requires a valid extended schema definition. Please go to the model's schema page in MOC, and generate and edit a valid schema for the input data\"))\n",
+    "display(HTML(\"You can view and edit the schema <a target=\\\"_blank\\\" href=\\\"\" + moc_url + \"#/models/model/\" + model[\"id\"] + \"/schema\\\">here</a>.\"))\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Examine example model assets\n",
+    "\n",
+    "Run the cells below, and we will attempt to locate the required model assets for you.  If specific\n",
+    "assets are not automatically recognized we will let you know what assets are available that could\n",
+    "potentially provide the needed data.  The asset role in ModelOp Center can then be changed to the appropriate type\n",
+    "such as BASELINE_DATA or COMPARATOR_DATA.  There is also an option to use the training data as the baseline data for\n",
+    "development purposes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "use_training_data_checkbox = widgets.Checkbox(value=False, description='Use training data as baseline data')\n",
+    "display(use_training_data_checkbox)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    },
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "\n",
+    "baseline_data_file = None\n",
+    "comparator_data_file = None\n",
+    "unknown_data_files = []\n",
+    "required_assets_df = None\n",
+    "required_assets_validations = []\n",
+    "\n",
+    "out = widgets.Output()\n",
+    "with out:\n",
+    "\tif (os.path.exists('required_assets.json')):\n",
+    "\t\tdisplay(HTML(\"</br>Here are the required assets for the dashboard model: \"))\n",
```

Comment:
Please update this to reference both, "dashboard model" and "standard risk test model"